analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -154,15 +153,13 @@ object PythonInspector : CommandLineTool {
             add("--json-pdt")
             add(outputFile)
 
-            if (requirementsFile != null) {
+            if (definitionFile.name == "setup.py") {
```

Comment:
Should "setup.py" be extracted to a constant to avoid redundancy?

Reply:
I think at least most other occurrences will be removed in follow-up changes, so I'd rather wait for that first.